tests/foreman/cli/test_subscription.py

```diff
@@ -281,3 +283,28 @@ def test_positive_Subscription_status_disabled(self):
             host_id = host[0].id
             host_content = entities.Host(id=host_id).read_raw().content
             assert "Disabled" in str(host_content)
+
+    @tier2
+    @pytest.mark.usefixtures("GoldenTicketHostSetup")
+    def test_positive_auto_attach_disabled_golden_ticket(self):
+        """Verify that Auto-Attach is disabled or "Not Applicable"
+        when a host organization is in Simple Content Access mode(Golden Ticket)
+
+        :id: 668fae4d-7364-4167-967f-6fc31ba52d26
+
+        :expectedresults: auto attaching a subscription is not allowed
+        and returns an error message
+
+        :BZ: 1718954
+
+        :CaseImportance: Medium
+        """
+        with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+            vm.install_katello_ca()
+            vm.register_contenthost(self.org_setup['label'], self.ak_setup['name'])
+            assert vm.subscribed
+            host = entities.Host().search(query={'search': 'name={}'.format(vm.hostname)})
+            host_id = host[0].id
+            with pytest.raises(CLIReturnCodeError) as context:
+                Host.subscription_auto_attach({'host-id': host_id})
```

Comment:
Shouldn't we (additionally) verify that on activation key?

Reply:
I do not think it is necessary for this test. If testing activation keys and golden ticket is needed, I will add that in a separate test.